vmd/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Vmd(Package):
+    """VMD provides user-editable materials which can be applied to molecular
+    geometry. These material properties control the details of how VMD shades
+    the molecular geometry, and how transparent or opaque the displayed
```

Comment:
```suggestion
    """VMD provides user-editable materials which can be applied to molecular
    geometry. 
    
    These material properties control the details of how VMD shades
    the molecular geometry, and how transparent or opaque the displayed
```
A first short paragraph is useful to extract a short description for e.g. module files.